### PR TITLE
[Clojure] add floating point numbers exercise

### PR DESCRIPTION
**Pull request by porkostomus**
A couple of the numbers were a tiny bit off from the f# version I forked this from. Most likely reasons:

1. I did something silly in my implementation and am losing precision somewhere.
2. BigDecimals in Java are slightly different from F#'s decimals.

### CODE REVIEW

File: languages/clojure/exercises/concept/floating-point-numbers/.docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+Different floating-point types can store different numbers of digits after the digit separator - this is referred to as its precision. This means that trying to store PI in a `single` will only store the first 6 to 9 digits (with the last digit being rounded).
```
Comment (bemself) [inline languages/clojure/exercises/concept/floating-point-numbers/.docs/introduction.md:3]: @porkostomus `store PI in a `single` `, sorry what does this 'single' mean, is there a data type called single? (I'm quite new to clojure...)

File: languages/clojure/exercises/concept/floating-point-numbers/src/floating-point-numbers.clj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+(ns floating-point-numbers)
+
+(defn interest-rate [balance]
+  (cond 
+    (< balance 0.0M) -3.213
+    (< balance 1000.0M) 0.5
+    (< balance 5000.0M) 1.621
```
Comment (bemself) [inline languages/clojure/exercises/concept/floating-point-numbers/src/floating-point-numbers.clj:7]: missing equals?

**Comment by ErikSchierboom** [comment]
@porkostomus Is there anything I can do to help with this PR?

File: languages/clojure/exercises/concept/floating-point-numbers/.docs/introduction.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/clojure/exercises/concept/floating-point-numbers/.docs/introduction.md:3]: I had the same question. The list of types below only mention `Double` and `BigDecimal`.

**Review by ErikSchierboom — APPROVED** [review]
Nice!

**Pull request merged**